txscript/hashcache.go

```diff
@@ -29,9 +31,17 @@ func NewTxSigHashes(tx *wire.MsgTx) *TxSigHashes {
 		HashPrevOuts: calcHashPrevOuts(tx),
 		HashSequence: calcHashSequence(tx),
 		HashOutputs:  calcHashOutputs(tx),
+		// HashUTXOS:    calcHashUtxos(tx),
+		HashUTXOS: chainhash.Hash{},
```

Comment:
Why did this change?

Reply:
It requires data from the utxoCache that does not exist there. We have to fill it elsewhere.

Reply:
But you replaced it with a value that isn't the correct one... why not an empty value until we can set it in the right place?